Factory.Cargo/TransportationFactory/AirTransportationFactory.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Factory.Cargo.Factory;
+using Factory.Cargo.Transportation;
+
+namespace Factory.Cargo.TransportationFactory;
+
+public class AirTransportationFactory : ITransportationFactory
```

Comment:
where are you using this Factory ?
i didn't find it.

Reply:
Fixed.